ServerCore/Helpers/NotificationHelper.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using ServerCore.DataModel;
+
+/// <summary>
+/// Content of a notification, intended to be broadcast to all players in a specific event, all players on a specific team, or a specific player.
+/// </summary>
+public class Notification
```

Comment:
I had put the set of types that go through signalr in ServerCore/ServerMessages/ServerMessageTypes.cs since the hub and listener need to know about them, but I don't have super strong feelings on that vs putting them near the components that use them. It might be nice to do something consistent though